### ISSUE TITLE
html_use_smartypants deprecated in Sphinx 1.6; use docutils.conf

### ISSUE THREAD

**Issue by stevepiercy** [rtfd/readthedocs.org#2940]
In Sphinx 1.6, `html_use_smartypants` is [deprecated](http://www.sphinx-doc.org/en/stable/config.html#confval-html_use_smartypants). To avoid typographical (curly) quotes, one must use a `docutils.conf` file at the same level as the `conf.py` file with the following setting:

```conf
[parsers]
smart_quotes: false
```

See also http://docutils.sourceforge.net/docs/user/config.html#smart-quotes

**Comment by stsewd** [comment]
I don't see that option on the current template https://github.com/rtfd/readthedocs.org/blob/master/readthedocs/doc_builder/templates/doc_builder/conf.py.tmpl

**Comment by stevepiercy** [comment]
@stsewd because of https://github.com/rtfd/readthedocs.org/blob/master/requirements/pip.txt#L6

**Comment by humitos** [comment]
@stevepiercy I'm not sure to understand the problem. Is this for the RTD docs itself? I mean, should we create a `docutils.conf` under our `docs/` directory with that content?

If that's the case, I don't see that we are using the old setting `html_use_smartypants` at https://github.com/rtfd/readthedocs.org/blob/b12e1885dc7f6da6b9719054e3654019b1e0fe65/docs/conf.py and the default is `True`. So, I'm confused about this issue/bug. Can you clarify?

Thanks!

**Comment by stevepiercy** [comment]
@humitos this is for all docs published at RTD in the event that RTD decides to bump Sphinx to 1.6.x or higher.

**Comment by humitos** [comment]
@stevepiercy in the previous comment you linked the `pip.txt` and that reqs file is for RTD itself. The `Sphinx` installed for docs built by RTD is pinned at https://github.com/rtfd/readthedocs.org/blob/b12e1885dc7f6da6b9719054e3654019b1e0fe65/readthedocs/doc_builder/python_environments.py#L223

and it's currently a `1.6.x`. Anyway, I don't see what we can do here since RTD didn't inject this setting in the past and that's a configuration that's up the user of RTD, we shouldn't force them.

I don't see too much that we can do here.

**Comment by stevepiercy** [comment]
@humitos, sorry, my mistake. I think this issue was a placeholder and reminder for our projects for the eventual update to Sphinx 1.6.x by RTD. I opened this issue when RTD was using Sphinx 1.5.x and we had to use `html_use_smartypants=False` to prevent quote conversion. We also did not want to pin our docs to a specific version of Sphinx and miss out on the eventual upgrade to RTD, so when building our projects in Travis-CI or locally, we would install 1.6.x and generate deprecation warnings. The `docutils.conf` file helped us work around that.

With the update to Sphinx 1.6.5 as shown by [git blame about 2 months ago](https://github.com/rtfd/readthedocs.org/blame/b12e1885dc7f6da6b9719054e3654019b1e0fe65/readthedocs/doc_builder/python_environments.py#L223), we can now remove `html_use_smartypants` from our `conf.py`, retain straight quotes without conversion, and not use the `docutils.conf` file.

I'm closing this issue.
